src/server/game/Groups/Group.cpp

```diff
@@ -278,6 +279,16 @@ void Group::ConvertToLFG(bool restricted /*= true*/)
     SendUpdate();
 }
 
+bool Group::CheckLevelForRaid()
+{
+    for (member_citerator citr = m_memberSlots.begin(); citr != m_memberSlots.end(); ++citr)
+        if (Player* player = ObjectAccessor::FindPlayer(citr->guid))
```

Comment:
FindConnectedPlayer is probably better. Iirc findplayer only checks the current map storage and only accounts for players currently in world (so no loading screen while teleporting from one place to another, for example)

Reply:
Used the same as Group::ConvertToRaid uses. But I can redo

Reply:
probably fine then